app/views/govuk_publishing_components/components/layout_header/_navigation_items.html.erb

```diff
@@ -1,7 +1,8 @@
+<%  navigation_items_label ||= "Top level" %>
 <% if navigation_items.any? %>
   <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
-  <nav class="gem-c-header__nav">
-    <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
```

Comment:
The default value for the aria-label is changing from `Top level navigation` to `Top level` - is that okay? Just want to check.

Reply:
Fair question! I thought it might come up. 
The reason is that when a `nav` has `aria-label`, AT will read it out as **"YOUR_ARIA_LABEL_TEXT navigation"**.  Therefore when I originally left the default value as `Top level navigation`, I noticed that VoiceOver was reading it out as "Top level navigation navigation".
Testing with NVDA a similar redundancy occurred –  when the screen reader reaches the landmark, it states that it is a nav, so having the word "navigation" in the `aria-label` text results in repetition.

Reply:
Good answer.